### DISCUSSION TITLE
The trace data found through the trace ID includes two microservices, with two Segments located, one for each microservice. However, the timing data for each only includes the duration of the respective microservice. Is this correct? Should it be the total duration?

### DISCUSSION THREAD

**Discussion by linshenfanying** [apache/skywalking#12320] [Q&A]
The trace data found through the trace ID includes two microservices, with two Segments located, one for each microservice. However, the timing data for each only includes the duration of the respective microservice. Is this correct? Should it be the total duration?

<img width="848" alt="Skywalking数据不对" src="https://github.com/apache/skywalking/assets/153265945/194c19cf-d9bc-4155-bd5d-06850a6e6807">

**Comment by wu-sheng** [comment]
In theory, this should not, as most HTTP is blocking access. But in the reality, it is possible, because it depends on how to measure the time, the data point of receiving the first bit or whole body transportation finished.

**Reply by linshenfanying** [reply]
So, under what circumstances would this situation usually occur?

**Reply by wu-sheng** [reply]
> But in the reality, it is possible, because it depends on how to measure the time, the data point of receiving the first bit or whole body transportation finished.

I explained it.
